ci-operator/step-registry/cucushift/upgrade/setchannel/cucushift-upgrade-setchannel-commands.sh

```diff
@@ -62,6 +65,7 @@ export PATH=${OC_DIR}:$PATH
 export DUMMY_TARGET="${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE}"
 # we need this DUMMY_TARGET_VERSION from ci config to download oc
 DUMMY_TARGET_VERSION="$(env "NO_PROXY=*" "no_proxy=*" oc adm release info "${DUMMY_TARGET}" --output=json | jq -r '.metadata.version')"
+export DUMMY_TARGET_VERSION
```

Comment:
we need to extract the minor version number here, or we should pass down the following `y_ver` to `extract_oc`, right?

Reply:
probably should bump `y_ver=$( echo "${DUMMY_TARGET_VERSION}" | cut -f2 -d. )` above `extract_oc` and then `if (( y_ver > 15 )); then`